src/kani-compiler/src/codegen_cprover_gotoc/codegen/statement.rs

```diff
@@ -505,7 +518,7 @@ impl<'tcx> GotocCtx<'tcx> {
         let loc = self.codegen_caller_span(&span);
         Stmt::block(
             vec![
-                self.codegen_assert_false(msg, loc),
+                self.codegen_assert_false(PropertyClass::SanityCheck, msg, loc),
```

Comment:
I think Celina is right and `codegen_fatal_error` should take a property class parameter.

The first thing I saw using it seemed to be an unsupported construct.

Reply:
oh yeah, let me fix that, I didn't include it in this commit.